dspace-api/src/main/java/org/dspace/content/CollectionServiceImpl.java

```diff
@@ -26,6 +26,7 @@
 import org.dspace.harvest.HarvestedCollection;
 import org.dspace.harvest.service.HarvestedCollectionService;
 import org.dspace.workflow.factory.WorkflowServiceFactory;
+import org.dspace.xmlworkflow.Role;
```

Comment:
Do we need this import? We should not use any workflow implementation specific classes.